src/uwtools/drivers/forecast.py

```diff
@@ -126,10 +131,64 @@ def resources(self) -> None:
         ???
         """
 
-    def run(self):
+    def run(
+        self,
+        config_file: str,
+        forecast_app: str,
+        forecast_model: str,
+        dry_run: bool = False,
+        log_file: Optional[str] = None,
+        machine: Optional[str] = None,
+        quiet: bool = False,
+        verbose: bool = False,
+    ) -> None:  # pragma: no cover
         """
         Runs FV3.
         """
+        # set up logging
+        name = f"{inspect.stack()[0][3]}"
+        log = cli_helpers.setup_logging(
+            log_file=log_file or "/dev/stdout",
+            log_name=name,
+            quiet=quiet,
+            verbose=verbose,
+        )
+        # read in the config file
+        # self.create_model_config(config_file=config_file)
+        experiment_config = config.YAMLConfig(config_file)
+        # prepare directories
+        run_directory = experiment_config["parameters"]["run_directory"]
```

Comment:
I'd expect that the section of the config file we'd want these settings to be a part of would be named something similar to `forecast`. Perhaps it's more specific with `FV3Forecast`?

Reply:
Hopefully handled better in the updated YAML now!